tests/fixtures/policies/scripted_policy.py

```diff
@@ -0,0 +1,36 @@
+class ScriptedPolicy:
+    """Simulates a garage policy object.
+
+    Args:
+        - scripted actions(list or dictionary): data structure indexed by
+            obervation, returns a corresponding action
+        - agent_env_infos(list or dictionary): data structure indexed by
+            obervation, returns a corresponding agent_env_info
+    """
+
+    def __init__(self, scripted_actions, agent_env_infos=None):
+        self.scripted_actions = scripted_actions
+        self.agent_env_infos = agent_env_infos
+
+    def set_param_values(self, params):
+        return params
+
+    def get_param_values(self):
```

Comment:
I don't think this follows the get_param_values contract. shouldn't it be a list or something?

Reply:
fixed